README.md

```diff
@@ -21,7 +21,7 @@
   - PNG (through [libspng](https://libspng.org/))
   - PPM
   - [StumpyCore](https://github.com/stumpycr/stumpy_core)
-  - WebP (through [libwebp](https://developers.google.com/speed/webp))
+  - WebP (through [libwebp](https://developers.google.com/speed/webp) and `libsharpyuv`)
```

Comment:
For some reason I couldn't figure out how to install `libsharpyuv` that's now required for webp :/ This meant I had to comment out the webp specs and webp portion of the benchmark while I was adding the pad method. Any suggestions on how I can install this dependency? I run linuxmint, a naive google search didn't yield me anything fruitful (I blame google for that).

Reply:
It should be included with WebP 1.3.0+.

Sorry I didn't account for older versions when updating, it just kinda worked on both Alpine and Arch.

One way to go might be adding a condition that checks the version and link with or without `libsharpyuv` depending on that.

Reply:
Given it's a part of WebP, maybe we shouldn't mention it in `README.md`?

Reply:
Yeah, that makes sense. I checked my own version of linuxmint (20.3) which follows Ubuntu's 20.04. `APT` is giving me problems right now and I can't seem to update the libwebp library locally :/ I've locally added a compiler flag `-Dlegacy_webp` which changes the `@Link` to not include `-lsharpyuv` so I'm able to compile, test, and run without commenting things out. I'll include it in my next commit, let me know if it's worth keeping.

Reply:
Can we mention this flag in `README.md`?

Reply:
Cool, if you're good with the flag, I'll add a note to the README :+1: